src/Controls/tests/DeviceTests/Elements/Entry/EntryTests.iOS.cs

```diff
@@ -36,5 +40,57 @@ int GetPlatformSelectionLength(EntryHandler entryHandler)
 
 			return -1;
 		}
+
+		[Fact]
+		public async Task ScrollSkipsAlert()
+		{
+			var entry = new Entry();
+
+			EnsureHandlerCreated(builder =>
+			{
+				builder.ConfigureMauiHandlers(handler =>
+				{
+					handler.AddHandler<VerticalStackLayout, LayoutHandler>();
+					handler.AddHandler<Entry, EntryHandler>();
+				});
+			});
+
+			var layout = new VerticalStackLayout
+			{
+				entry
+			};
+
+			layout.WidthRequest = 300;
+			layout.HeightRequest = 800;
+
+			await InvokeOnMainThreadAsync(async () =>
+			{
+				var contentViewHandler = CreateHandler<LayoutHandler>(layout);
+				await contentViewHandler.PlatformView.AttachAndRun(async () =>
+				{
+					var layoutPlat = layout.ToPlatform();
+					var win = layoutPlat.Window;
+					if (win is UIWindow uIWindow)
+					{
+						var alert = UIAlertController.Create("Popup Alert", "This is a popup", UIAlertControllerStyle.Alert);
+
+						alert.AddTextField((textfield) =>
+						{
+							textfield.Placeholder = "Placeholder Text";
+						});
+
+						_ = uIWindow.RootViewController.PresentViewControllerAsync(alert, true);
```

Comment:
I don't think we want to ignore past these Async calls. 

My thoughts for this test
1) await these calls
2) make this test part of a Collection. You can see where this is done a few places with an inner class

[Category(TestCategory.Accessibility)]
		[Collection(ControlsHandlerTestBase.RunInNewWindowCollection)]
		public class InNewWindowCollection : ControlsHandlerTestBase
		{

3) use entry.GetBoundingBox to validate that the location of the entry hasn't shifted

Reply:
I don't think we want to ignore past these Async calls. 

My thoughts for this test
1) await these calls
2) make this test part of a Collection. You can see where this is done a few places with an inner class

[Category(TestCategory.Accessibility)]
		[Collection(ControlsHandlerTestBase.RunInNewWindowCollection)]
		public class InNewWindowCollection : ControlsHandlerTestBase
		{

3) use entry.GetBoundingBox to validate that the location of the entry hasn't shifted